web/gds-user-ui/src/application/store/stepper.slice.ts

```diff
@@ -37,22 +38,41 @@ const stepperSlice: any = createSlice({
       state.currentStep = payload.currentStep;
     },
     incrementStep: (state: any) => {
+      // always set isDirty to false when incrementing step
```

Comment:
@daniellemaxwell  When the user is filling out the form, the `isDirty state` is set to true as soon as they click on the next button we need to change it to false. The purpose of adding isDirty to the state is to improve the handling of how the user navigates between steps using the certificate label component.

By setting isDirty to true when the user starts filling out the form, we can track any changes made. This is important because when the user jumps to another step without saving their changes, they might lose their data. Therefore, we need to detect if any changes have been made in the form and display a message to inform the user that they should save their changes.

To achieve this, we can update the isDirty state to false when the user clicks on the next button. This way, we reset the isDirty state and indicate that no changes have been made since the last save.

Reply:
So, if a user goes back to a step that they previously completed `isDirty` will automatically set to `true`? Then if they make changes, like deleting info, `isDirty` will be set to `false` after they click `Next`?

Reply:
Keep in mind  that the `isDirty state` is specifically used to manage label stepper components. When a user navigates back to a previously completed step, the isDirty state is initially set to false. However, once they start typing, it is set to true. The isDirty state changes whenever there is any modification in the form. After the user clicks on the "Next" or "Previous" button, the isDirty state is set back to false.